components/level/level.cpp

```diff
@@ -50,11 +61,20 @@ namespace Level
         Serial::ScopedCategorySaver cat("Level", saver);
 
         saver.save(mTilesetCelPath);
+        saver.save(mSpecialCelPath);
```

Comment:
These save files seem a bit flaky, adding this new value likely breaks compatibility.
Probably should be using key value pairs at least?
#24